spec/fixtures/locations.json

```diff
@@ -14,6 +14,7 @@
       "properties": {
         "title": "Belfast Citizens Advice Bureau",
         "address": "Merrion Business Centre, 58 Howard St, Belfast, BT1 6PJ",
+        "booking_centre": "928543c7-e8f0-45b2-a300-c2f996b007d9",
```

Comment:
It exposes `"booking_centre": "Greendale Citizens Advice"` so this change is coupled to the GeoJSON changing too. Hanging off `booking_location_id` lets us retire `booking_centre` down the line.